tests/threshold/threshold-rule-flow/README.md

```diff
@@ -0,0 +1,3 @@
+PCAP
+====
+PCAP from https://wiki.wireshark.org/Internet_Control_Message_Protocol
```

Comment:
Could you add the purpose of this test ?

Reply:
Not really seeing the point. Test name says it all.

Reply:
I think there is a generic template for readme now, with also redmine ticket involved... cc @jufajardini ?

Reply:
Indeed, when one uses createst, it generates a readme file that expects: Description, Pcap, and Ticket/related issues). I personally find it useful to add something more descriptive in case people use SV tests to understand how something works, or as examples.